### DISCUSSION TITLE
When to run dbt_artifacts models

### DISCUSSION THREAD

**Discussion by seub** [brooklyn-data/dbt_artifacts#214] [Q&A] [answered]
Hello, I have two small questions or clarifications about dbt_artifacts version 2.0.0:

1. Once I have done one `dbt run` in my project that includes the dbt_artfacts models, it is completely unnecessary in subsequent runs to include dbt_artifacts models. (That is, unless I upgrade the package dbt_artifacts). Is this correct?

2. I should be cautious not to include dbt_artifacts models when I do `dbt_run --full-refresh` in my project, otherwise I will lose all previous artifacts. Is this correct?

Thanks!

**Comment by jaypeedevlin** [comment, marked as answer]
Hey Brice!  Appreciate you dropping the question here

1.  You're correct, although I would usually recommend having them 'run' as part of your DAG if you can so that you have less manual steps involved
2.  Yes. This is a **fantastic** observation, and I've just opened issue #215 so that we can fix this, because it's definitely behavior we want to prevent (and the current setup makes this way too easy IMO).

**Reply by seub** [reply]
Thank you very much for your answers! 

For 2. it is indeed easy to fix with the `full_refresh` config. I added in my dbt_project.yml:
```yaml
models:
  my_project:
    ...
  dbt_artifacts:
    ...
    +full_refresh: false
```
